homepage/index.html

```diff
@@ -0,0 +1,68 @@
+            .';dXx.   '0MMM0'                            '0Wo                                  
+           .;cll:.    .dWMMWo                            .kXc                                  
+                       :XMMMK,                     .lxxdc....                                  
+                       .OMMMWd.                   .dWkcOWo                                     
+                        oWMMMK;                   .xWx,kWd.                                    
+                        ;XMMMWx.            .,:lc. ,kOkOx'                                     
+                        .kMMMMX:            ..cXWo   ...                                       
+                         lWMMMMk.             '0Wo  .:c;.                                      
+                         ,KWKO0Kc             .kNl.cXKkKK:                                     
+                         .kXc 'Ok.    .oko;.   .,..kWd'xWx.                                    
+.                           cX0xONNc     .cx00xo:.   lX0xKXc                     ..     .....    
+,dO00kc.                       '0MN00NO'    .:l:coxO0kl,.,:c:'                   ,dO00kl;lx0KXX0x;. 
+XMMMMMNc                       .xWx..dXl    lNMNOocc:lx0Od:.                    ;KMMMMMWWWWMMMMMMWx.
+WMMMMMWo                        :XKxdKW0'  .xMMMMMMWKxc:cdO0kl'                 cNMMMMMM0c:xNMMMMMWx
+WMMMMMWd. .',,..                'OMWXKNWo  .kMMMMMMMMMWXkoc:ld:.  ..',,,'..     lNMMMMMWd  .kMMMMMMX
+WMMMMMM0xOXNWWN0o'               dW0;.cXK,  dWMMMMMMMMMMMMNo.  'lx0XNWWWNX0xc.  'OWMMMMWo  .dWMMMMMW
+WMMMMMMMWNWMMMMMMK:              ;XXxlkNWd. ;XMMMMMMMMMMMWx. .dXMMMMMNKNMMMMWXo. 'kWMMMWd  .kMMMMMMX
+WMMMMMM0:'lKMMMMMMK,             .dNMMMMMK; .xWMMMMMMMMMMO' 'OWMMMMMK:.cXMMMMMWk. ,0MMMM0l:xNMMMMMWx
+WMMMMMMd   oWMMMMMWl  .,cllollc;.  ,dNMMMWx. ,0MMMMMMMMMWo  oWMMMMMWd. .xMMMMMMNc .dWMMMMMWMMMMMMNx.
+WMMMMMWd   cNMMMMMWl  oWMMNNWMMWXk:  cXMMMX:  cXMMMMMMMMNl .xMMMMMMWl   dWMMMMMWo  oWMMM0dk0KKKOd;. 
+NMMMMMWx. .xWMMMMMK; .kMWO;,xWMMMMX: .dWMMWk. .oWMMMMMMMWd. lNMMMMMWx. .kMMMMMMX: .xMMMWd.  ...     
+kWMMMMMXxlkNMMMMMXc  lNMK; .lNMMMMWo  oWMMMNl  ,KMMMMMMMMK; .dNMMMMMNd:xNMMMMMNo. :XMMMWo           
+.oKWMMMMMMMMMMWKd' .lXMMNOkOXMMMWKd. 'OMMMMMNxcdNMMMMMMMMM0;  :kXWMMMMMMMMMWXx;  ;KMMMMK,           
+.;ldkOOOOkdl;. .:kWMMMWOlcccc:,'.  .xWMMMMMMMMMMMMMMMMMMM0'   ':oxkOOOkxo:.    .lxkxl'            
+            :XMMMMMMKdc;;:cok0d. ;KMMMMMMMMMMMMMMMMMMWx.                                       
+            .oXMMMMMMMMWWMMMWKc  '0WMMMMMMMMMMMMMMMMWk'                                        
+              'oOXWMMMMMMWXOl.    .lOXWMMMMMMMMMMWKx:.                                         
+                .;ok0KK0ko;.        .;d0XWMMWWNKxc'                                           
+</pre>
+            </div>
+        </div>
+
+        <div class="container">
+            <h1>An efficient, schema-based binary serialization format</h1>
```

Comment:
Can you change this to An extremely simple, fast, efficient, cross-platform serialization format